examples/ffs-service-example.c

```diff
@@ -59,6 +59,8 @@
 
 /******************** Endpoints handling *******************************/
 
+extern int gd_get_ep_by_nmb(int nmb);
```

Comment:
Good catch but instead of declaring this function manually just include ffs-daemon.h and fix it by adding a static keyword to function declaration;)

Please also notice that this functionality is a little but outdated since we managed to integrate this into systemd (https://www.freedesktop.org/software/systemd/man/systemd.socket.html - ListenUSBFunction). So this should be dropped and replaced with suitable API calls to systemd.